rpc/flipt/auth/auth.proto

```diff
@@ -58,7 +58,7 @@ enum Method {
   METHOD_KUBERNETES = 3;
   METHOD_GITHUB = 4;
   METHOD_JWT = 5;
-  METHOD_CLOUD = 6;
+  METHOD_CLOUD = 6 [deprecated = true];
```

Comment:
chose to just deprecate instead of remove because it could break consumers at compile time: https://buf.build/docs/breaking/rules#enum_no_delete